libs/common/src/ui-scripts/disableFormOnSubmit.js

```diff
@@ -0,0 +1,32 @@
+// Prevent multiple form submissions by pressing enter or clicking the submit button
+// The govukButton component has a preventDoubleClick property, but this only debounces in a one-second window,
+// and is therefore not sufficient for our needs as multiple forms take longer than a second to submit.
+
+const addDisableFormOnSubmit = () => {
+  const forms = document.querySelectorAll('form');
+  // only take the last form on the page, as it is foreseeable to have multiple forms on a page,
+  // and the 'continue' button should always be in the last form on a page.
+  const lastForm = forms[forms.length - 1];
```

Comment:
Need to ensure `forms.length` is not zero.

Reply:
Sounds good, I'll do this so it's clearer (as if `querySelectorAll` returns an empty array, `  const lastForm = forms[forms.length - 1];` would equal `undefined` that would be picked up on line 11, but I agree this can be clearer.